serialization/json.md

```diff
@@ -1,26 +1,85 @@
 # JSON Serialization
 
-Ideally JSON will be very similar (or even the same) to JSON-LD.
+As described in [Logical Values](logical/README.md), JSON is a general purpose data format
```

Comment:
Your "logical" serialization should not be a dependency of the json serialization. The json one should be self sufficient and independent.

Reply:
"Logical values" exist whether they are documented or not.  An application that de-serializes data will have some in-memory state as a result.  We don't want that state to have any dependency on which serialization format was used - strange, difficult-to-diagnose problems could occur: "the application did this when I read a JSON SBOM, but did something else after reading the same SBOM in RDF"

I anticipated that the application state would be discussed and agreed to before we define any of the serializations, hence the wording.  But the preface clause is unnecessary for the JSON serialization and can be deleted.